pkg/eventlog/mongo/mongo.go

```diff
@@ -108,3 +115,35 @@ func (m *Log) Clear(id string) error {
 	_, err := c.RemoveAll(bson.M{"_id": bson.M{"$lte": id}})
 	return err
 }
+
+func (m *Log) newSession() (*mgo.Session, error) {
+	tlsConfig := &tls.Config{}
+
+	dialInfo, err := mgo.ParseURL(m.url)
+	if err != nil {
+		return nil, err
+	}
+
+	dialInfo.Timeout = env.MongoConnectionTimeoutSecWithDefault(1)
+
+	if m.certPath != "" {
+		roots := x509.NewCertPool()
```

Comment:
Just a question, not a request, but I'm wondering if we are going to want to allow people to bring their own custom CA's too eventually?

Reply:
i haven't thought about this. can you please elaborate, as you have more experience in this field definitely, why would this be a wanted feature. I haven't worked with custom CA so far. is it about people being paranoid?

Reply:
Lots of companies have their own CA, and it's not a trusted root by default. It lets them issue certs inside their own infra without dealing with paying for it, or the rotation schedules demanded by Lets Encrypt.

Another reason is self-signed certs. We can let people specify their ca and use a self-signed cert without having skip verify turned on. (which addresses my comment below)

Reply:
I'm fine with keeping skip verify on for now, and then circling back in a follow-on PR to support custom CAs. I don't want to over-complicate this PR, what you have works just fine as an intermediate step. 👍

Reply:
:shipit:

Reply:
added a todo and created #540